docs/sdkx-react-native/notifications-ios.mdx

```diff
@@ -237,6 +237,20 @@ If you do not want the in-app notifications to support provided by the
 Helpshift SDK, please set this flag to `false`. The default value of this
 flag is `true` i.e in-app notifications will be enabled.<br />
 
+### Pausing In-app notifications {#pause-in-app}
+
+If you have enabled in-app notifications, use the API `pauseDisplayOfInAppNotification:` to pause/resume the notifications. When `true` is passed to this method display of in-app notifications is paused even if they arrive. When you pass a `false`, the in-app notifications start displaying.
```

Comment:
- Why is there a `:` after API name on this line?
- When `true` is passed to this method, display of in-app notifications....
- When you pass `false`, the in-app.....

Fix in other pages as well if needed.